### ISSUE TITLE
Bring back mono debug symbols

### ISSUE THREAD

**Issue by haf** [xyncro/chiron#34]
It was removed in the commit to re-create the nuget

**Comment by kolektiv** [comment]
Yup, this should definitely go back in, I'll fix that (or if you get there first, I'll accept a PR! ;) ) Only got removed as NuGet seemed to be being odd with file paths and path separators... (Grrrrr, NuGet)

**Comment by ctaggart** [comment]
6.3.0 enables SourceLink and includes the portable pdb files. That is the new format that works on mono too. This can be closed.
